lib/src/rules/prefer_single_quotes.dart

```diff
@@ -9,7 +9,7 @@ import 'package:meta/meta.dart';
 import '../analyzer.dart';
 
 const _desc =
-    r"Prefer single quotes where they won't require escape sequences.";
+    r"Only use double quotes for strings containing single quotes.";
```

Comment:
Make this single quotes while you're here?

😄

Reply:
Done.